lib/cn/kvset.c

```diff
@@ -3711,8 +3717,8 @@ kvset_from_iter(struct kv_iterator *iv)
 void
 kvset_maxkey(struct kvset *ks, const void **maxkey, u16 *maxklen)
 {
-    *maxkey = ks->ks_minkey;
-    *maxklen = ks->ks_minklen;
+    *maxkey = ks->ks_maxkey;
```

Comment:
Nice catch :-)

Reply:
Hehe, wasn't a problem since we had zero callers :)